heaps/min_heap.py

```diff
@@ -10,67 +10,105 @@ def __str__(self):
     def __repr__(self):
         return str(self.value)
 
+
 class MinHeap:
 
     def __init__(self):
         self.store = []
 
-
-    def add(self, key, value = None):
+    def add(self, key, value=None):
         """ This method adds a HeapNode instance to the heap
             If value == None the new node's value should be set to key
-            Time Complexity: ?
-            Space Complexity: ?
+            Time Complexity: Ologn
+            Space Complexity: On
```

Comment:
✨ The space complexity here is determined by the recursive call stack of `heap_up` which is just O(log n)